ot/dr.py

```diff
@@ -173,7 +183,7 @@ def cost(P):
             for j, xj in enumerate(xc[i:]):
                 xj = np.dot(xj, P)
                 M = dist(xi, xj)
-                G = sinkhorn(wc[i], wc[j + i], M, reg, k)
+                G = sinkhorn(wc[i], wc[j + i], M, reg/regmean[i,j], k)
```

Comment:
the reg term should be proportional to the distance (since there is a division when computing the K matrix in sinkhorn)

Reply:
```suggestion
                G = sinkhorn(wc[i], wc[j + i], M, reg*regmean[i,j], k)
```